lib/query.js

```diff
@@ -35,7 +35,7 @@ class Query {
 
 		return this.model.dbCollection()
 			.then(collection => this.mquery.collection(collection).findOne())
-			.then(doc => doc ? new this.model(doc) : null); // eslint-disable-line new-cap
+			.then(doc => doc ? new this.model(doc) : null); // eslint-disable-line babel/new-cap, new-cap
```

Comment:
`babel/new-cap` shouldn't be necessary, try updating your `xo` via `npm update`.